### DISCUSSION TITLE
Did multiple default settings change with version 0.12.0?

### DISCUSSION THREAD

**Discussion by mawkler** [neovide/neovide#2253] [General]
I just upgraded to version 0.12.0, and it seems that multiple settings changed. The animations are significantly slower (I'm using the default settings). Also, my font (FiraCode) suddenly looks _way_ larger, and I had to change the font size from `h11` to `h7` for it to look as it used to.

I'm not sure if it's the animation speed, but it also feels way slower when moving the cursor around. I tried replicating the previous animation speed, but when comparing with version 0.11.2 I feel a delay when pressing keys in 0.12.0.

I couldn't find any mention of this anywhere.

**Comment by fredizzimo** [comment]
> The animations are significantly slower

The actual length total length of the animation should not have changed much, but the acceleration profile is different. More specifically the current animation works like a critically damped spring that reach 98% of the destination in scroll_animation_length time. 

Previously, this was used https://easings.net/#easeOutExpo, which get almost there very quickly.

The problem with that was, that it can't handle repeating key presses without constant acceleration/deceleration, which is very jarring when the animation is actually smooth, so another method had to be used. 

> Also, my font (FiraCode) suddenly looks way larger, and I had to change the font size from h11 to h7 for it to look as it used to.

I believe that you are seeing this bug 
* https://github.com/neovide/neovide/issues/2214

With this fix waiting to get merged
* https://github.com/neovide/neovide/pull/2218

> I'm not sure if it's the animation speed, but it also feels way slower when moving the cursor around. I tried replicating the previous animation speed, but when comparing with version 0.11.2 I feel a delay when pressing keys in 0.12.0

That one, I'm not sure about, if anything, the response times should now be faster. But maybe there's some kind of other bug affecting both animation speeds. Perhaps it's detecting the wrong refresh rate here https://github.com/neovide/neovide/blob/bbb26e84ac0d76a44ec71c53f7e104bc37054b9a/src/window/update_loop.rs#L146

**Reply by mawkler** [reply]
@fredizzimo I see. Thank you for the reply!
